### DISCUSSION TITLE
SpiralLinux 11.220925 release

### DISCUSSION THREAD

**Discussion by geckolinux** [SpiralLinux/SpiralLinux-project#131] [Announcements]
![SpiralLinux-transparent-small](https://user-images.githubusercontent.com/2692138/172185018-77411974-5e2b-43b6-aed8-42f492047cd6.png)

### Download: https://spirallinux.github.io

The SpiralLinux project is pleased to announce the 11.220925 release with some fixes and improvements. Many thanks to all the users that reported issues and suggested improvements! SpiralLinux offers individual live installable ISO images for the Cinnamon, XFCE, Gnome, Plasma, Mate, Budgie, and LXQt desktop environments.

## Changes to all editions

- Built from the latest Debian Stable 11.5 release. Users with prior installations of SpiralLinux can [simply update](https://github.com/SpiralLinux/SpiralLinux-project/wiki#updates-within-a-debian-stable-release-lifecycle) their system to receive the 11.5 release directly from Debian
- Firefox ESR from Debian Stable now at version 102.3
- Smoother two-finger scrolling with touchpads enabled in Firefox via environment variable
- Realtek r8168 WiFI theoretically supported out-of-the-box. _Testing and reporting by users with any Realtek WiFI hardware would be appreciated._
- Substituted newer `firmware-iwlwifi` package from Debian Testing to fix major connection problems with the Debian Stable firmware version on AX20x Bluetooth hardware
- Bitmap fonts disabled to avoid poor font rendering, such as in HTML emails
- SecureBoot support. ***Note:*** _SecureBoot support may not work out-of-the-box on all systems due to variations and quirks in different UEFI implementations. SecureBoot may not be desirable for some users due to the [limitations and complications](https://wiki.debian.org/SecureBoot#Secure_Boot_limitations) that it brings by design, and it may be preferable to disable SecureBoot in the device's UEFI settings_
- Installation possibly supported on 64-bit systems with 32-bit UEFI implementations. _Testing and reporting would be appreciated._
- Fixed `failed to load canberra-gtk-module` in `.xsession-errors`
- Disabled annoying beep when booting live system in EFI mode
- Preinstalled `alsa-ucm-conf` package for Chromebook chtmax98090 soundcards

### Changes to Budgie edition
- Added missing AppIndicator applet for system tray functionality with some applications
- Added missing `gnome-online-accounts` package

### Changes to Cinnamon edition
- Added missing Catfish search utility (included version of Nemo is good for finding filenames but does not support searching in file contents)

### Changes to Gnome edition
- Fixed an issue that would cause some systems to hang when rebooting after Calamares installation

### Changes to Mate edition
- Workaround for some preinstalled applications in the menu needing root privileges that were failing to launch

## About SpiralLinux

SpiralLinux is a selection of Linux spins built from Debian GNU/Linux, with a focus on simplicity and out-of-the-box usability across all the major desktop environments. SpiralLinux serves as an alternative live installation method for a highly reliable customized Debian system using only official Debian package repositories. Please visit the official [SpiralLinux website](https://spirallinux.github.io/) to learn more about its unique features and reasons for existence.

### What will happen to GeckoLinux?

GeckoLinux will continue to be maintained. However, SUSE and openSUSE have announced some major changes that will eventually affect the Leap branch a few years from now, so SpiralLinux would fill the void in the event that openSUSE Leap ceases to exist or turns into a completely different sort of product.

<hr /><em>This discussion was created from the release <a href='https://github.com/SpiralLinux/SpiralLinux-project/releases/tag/11.220925'>SpiralLinux 11.220925 release</a>.</em>

**Comment by artytux** [comment]
Hello
Installed SpiralLinux on Thursday 29-09 and it's a pleasure to drive very very nice.
Reading the news in DistroWatch and saw you would like some feedback well I can tell you that Realtek r8168 WiFi picked on the available and also it works fine
Didn't get  WiFi with GeckLinux 15 months good distro noticed after install not that it mattered I don't use my WiFi many many years since it was causing music playback problems long ago.
Worth checking thou for SpiralLinux and not a problem Hope it helps

Regards
flyingwithtux

**Reply by geckolinux** [reply]
>  Reading the news in DistroWatch and saw you would like some feedback well I can tell you that Realtek r8168 WiFi picked on the available and also it works fine

Wow, that's great news. Thanks a lot for the report!
I've personally considered several laptops with Realtek WiFI that were offered at a really good price, but I later decided against them due to the poor reports about Realtek WiFI, especially on Linux. Really glad that it was quite easy to include support for it out-of-the-box in SpiralLinux, thanks to Debian for packaging the driver.

**Comment by artytux** [comment]
Hello Again
I'm having to dis-associate from my flyingalone name found who is using that mmm, that's how flyingwithtux is me now on, have to remember on DW,
In answer to you're Q about the Realtek Model I'e found this, oh if you need more info just ask, glad to help

fa@fasystem:~$ lspci
00:00.0 Host bridge: Intel Corporation 8th Gen Core Processor Host Bridge/DRAM Registers (rev 07)
00:02.0 VGA compatible controller: Intel Corporation CometLake-S GT2 [UHD Graphics 630]
00:14.0 USB controller: Intel Corporation Cannon Lake PCH USB 3.1 xHCI Host Controller (rev 10)
00:14.2 RAM memory: Intel Corporation Cannon Lake PCH Shared SRAM (rev 10)
00:16.0 Communication controller: Intel Corporation Cannon Lake PCH HECI Controller (rev 10)
00:17.0 SATA controller: Intel Corporation Cannon Lake PCH SATA AHCI Controller (rev 10)
00:1b.0 PCI bridge: Intel Corporation Cannon Lake PCH PCI Express Root Port #21 (rev f0)
00:1c.0 PCI bridge: Intel Corporation Cannon Lake PCH PCI Express Root Port #5 (rev f0)
00:1c.6 PCI bridge: Intel Corporation Cannon Lake PCH PCI Express Root Port #7 (rev f0)
00:1d.0 PCI bridge: Intel Corporation Cannon Lake PCH PCI Express Root Port #9 (rev f0)
00:1d.2 PCI bridge: Intel Corporation Cannon Lake PCH PCI Express Root Port #11 (rev f0)
00:1d.3 PCI bridge: Intel Corporation Cannon Lake PCH PCI Express Root Port #12 (rev f0)
00:1f.0 ISA bridge: Intel Corporation Device a308 (rev 10)
00:1f.3 Audio device: Intel Corporation Cannon Lake PCH cAVS (rev 10)
00:1f.4 SMBus: Intel Corporation Cannon Lake PCH SMBus Controller (rev 10)
00:1f.5 Serial bus controller [0c80]: Intel Corporation Cannon Lake PCH SPI Controller (rev 10)
03:00.0 Network controller: Realtek Semiconductor Co., Ltd. RTL8192EE PCIe Wireless Network Adapter
05:00.0 PCI bridge: ASMedia Technology Inc. ASM1083/1085 PCIe to PCI Bridge (rev 04)
07:00.0 Ethernet controller: Realtek Semiconductor Co., Ltd. RTL8111/8168/8411 PCI Express Gigabit Ethernet Controller (rev 15)`

Regards
flyingwithtux (not the other fly (humor))

**Reply by geckolinux** [reply]
;-) Thanks!

> `RTL8192EE`

Noted.
